commit 230c3f0139f74f5c758e8a75a1eff941b1fb8ba0
Author: Ben Webb <ben@salilab.org>
Date:   Fri May 21 16:16:08 2021 -0700

    Don't use deprecated SetTotalBytesLimit()
    
    The protobuf folks changed the signature of
    SetTotalBytesLimit() in 3.6, so use this new
    function if we're building with a sufficiently
    new protobuf. This quiets a deprecation warning.

diff --git a/modules/npctransport/src/SimulationData.cpp b/modules/npctransport/src/SimulationData.cpp
index 08a31e7..7812cc2 100644
--- a/modules/npctransport/src/SimulationData.cpp
+++ b/modules/npctransport/src/SimulationData.cpp
@@ -141,7 +141,11 @@ void SimulationData::initialize(std::string prev_output_file,
   if(fd!=-1){
     google::protobuf::io::FileInputStream fis(fd);
     google::protobuf::io::CodedInputStream cis(&fis);
+#if GOOGLE_PROTOBUF_VERSION >= 3006000
+    cis.SetTotalBytesLimit(500000000);
+#else
     cis.SetTotalBytesLimit(500000000,200000000);
+#endif
     read= pb_data.ParseFromCodedStream(&cis);
     IMP_C_CLOSE(fd);
   }
diff --git a/modules/npctransport/src/protobuf.cpp b/modules/npctransport/src/protobuf.cpp
index 93bc1de..2f3d323 100644
--- a/modules/npctransport/src/protobuf.cpp
+++ b/modules/npctransport/src/protobuf.cpp
@@ -551,7 +551,11 @@ bool load_output_protobuf
   if(fd!=-1) {
     google::protobuf::io::FileInputStream fis(fd);
     google::protobuf::io::CodedInputStream cis(&fis);
+#if GOOGLE_PROTOBUF_VERSION >= 3006000
+    cis.SetTotalBytesLimit(500000000);
+#else
     cis.SetTotalBytesLimit(500000000,200000000);
+#endif
     is_ok=output.ParseFromCodedStream(&cis);
     IMP_C_CLOSE(fd);
   }
